backend/src/core/mail/application/mail-service.ts

```diff
@@ -0,0 +1,17 @@
+import { inject, injectable } from 'inversify';
+import { MailType } from '~/shared/enums/enums';
+import { CONTAINER_TYPES, MailProps } from '~/shared/types/types';
+import { MailRepository } from '~/core/mail/port/mail-repository';
+
+@injectable()
+export class MailService {
```

Comment:
As I understood to send mail you don't require to use MailService, just enough MailRepositoryAdapter, or is it some example? If it usage example please add a comment before the class declaration.